src/view/com/composer/select-language/SuggestedLanguage.tsx

```diff
@@ -19,10 +19,16 @@ import {Text} from '#/components/Typography'
 const onIdle = globalThis.requestIdleCallback || (cb => setTimeout(cb, 1))
 const cancelIdle = globalThis.cancelIdleCallback || clearTimeout
 
-export function SuggestedLanguage({text}: {text: string}) {
+export function SuggestedLanguage({
+  text,
+  fallback,
+}: {
+  text: string
+  fallback?: string
+}) {
   const [suggestedLanguage, setSuggestedLanguage] = useState<
     string | undefined
-  >()
+  >(fallback)
```

Comment:
nit, if we're doing the other change

```suggestion
  >(text.length === 0 ? replyToLanguage : undefined)
```